stable/coredns/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+name: coredns
+version: 0.1.0
+description: CoreDNS is a DNS server that chains middleware and provides Kubernetes DNS Services
+keywords:
+- coredns
+- dns
+- kubedns
+home: https://coredns.io
+icon: https://coredns.io/img/coredns-logo.png
```

Comment:
There is a github repo for this, github.com/coredns/logo that is waaaay more stable than this will be